### ISSUE TITLE
`@@index()` doesn't support `type`

### ISSUE THREAD

**Issue by he-la** [zenstackhq/zenstack#390]
**Is your feature request related to a problem? Please describe.**
Prisma supports setting an index type for PostgreSQL databases, e.g. `@@index([field1, field2], type: BTree)` ([docs](https://www.prisma.io/docs/concepts/components/prisma-schema/indexes)).

Zenstack complains with `Attribute "@@index" doesn't have a parameter named "type" []`

**Describe the solution you'd like**
Zenstack should be fully compatible with the prisma schema.

**Describe alternatives you've considered**

**Additional context**
- ZenStack version: 1.0.0-alpha.113
- Prisma version: 4.11.0
- Database type: Postgresql

**Comment by ymc9** [comment]
Good catch! It seems several attribute parameters are missing.

**Comment by ymc9** [comment]
If it's blocking, you can use the `@@prisma.passthrough` as a temporary workaround.

```
model User {
  @@prisma.passthrough('@@index([field1, field2], type: BTree)')
}
```

**Comment by he-la** [comment]
Thanks!

**Comment by ymc9** [comment]
I'll include a fix in the next release 😄. PR is also welcome!

**Comment by he-la** [comment]
I'm currently trying to migrate our production to zenstack. Will try to find some time to PR once I have confidence that ZenStack will work for us :rocket:

**Comment by ymc9** [comment]
Sure. Please keep the issues coming, and let us know which are blocking you so we can fix them with a higher priority.

**Comment by ymc9** [comment]
Hi @he-la , this issue is resolved by the latest alpha.114 release.

**Comment by ymc9** [comment]
closed by #400
